### DISCUSSION TITLE
为啥普通用户只能建立2条通道了

### DISCUSSION THREAD

**Discussion by hotsmile** [natfrp/wiki#102] [想法]
难道是缺钱了吗？  2条通道真心干不了啥，这未免有点太抠了，等同于鸡肋！！ ，普通用户活跃度大大降低呀

[a comment by an automated account is not shown]

**Comment by DyNooob** [comment]
的确有点反人类。建站一个http一个https就占满了，ssh不能再建隧道。。。
而且最近建站想起自己在樱花有账号了，就登上看了看，原来几百个G流量全没了，每个月只有10G基础
没说樱花不好，不过原来每天打卡签到好不容易攒下来的流量一下子过期了，真的挺难受

**Comment by Evdgod** [comment]
免费已经不错了，我寻思一个月10块也不贵啊

**Reply by Yue-cn** [reply]
“一个月10块也不贵啊”，为什么SakuraFrp要有许多人呢？因为打着"免费"口号。
又不是人人都是上海生的。。。

**Reply by Evdgod** [reply]
樱花是一开始打着免费的口号，里面的功能随意使用吗，非常抱歉，我不太了解樱花的历史，可能说的不太正确。我只是用云服务器的价格和樱花的现价格比较的

**Reply by Yue-cn** [reply]
* 具体看这张表
![image](https://user-images.githubusercontent.com/79188080/218248898-b167d6cf-ff4f-4b31-b18d-4d8b10fa23a6.png)

**Reply by Evdgod** [reply]
这张图是不是指在免费时间内用户数激增？是不是因为用户过多，维护建点的费用增加所以收费的吧？感觉所有项目都是这样啊，一开始人数少，开销不大还可以提供尽可能好的服务，人数多了，各项压力也就大了，收费似乎是必然之举。。。

**Comment by nothesong** [comment]
我觉得免费用户1个就可以，为爱发电走不远

**Reply by shenyangyeshuai** [reply]
1个就真有可能少有人用了。因为2个隧道，一个 ssh 可以做调试。如果只有一个隧道，出个问题都没法 ssh 远程登录解决，这个可用性就大大降低了。

**Comment by shenyangyeshuai** [comment]
樱花的历史不了解，但是从经济的角度看，免费的东西，而且速度/稳定性还可以接受，保证基本能用，这就算很良心了。

通道数量的问题，我觉得可以 10 块钱买一个青铜，把上限提高到 10 ，把该建的隧道都建好、测好，然后回到免费等级后，保留一个 ssh 常开，其它的隧道，用的时候现开，不用就关了，也不会被删。
![image](https://user-images.githubusercontent.com/3853749/223299948-7a7523de-c76d-4409-a855-3d30ead3cc80.png)
至于 10 块钱的问题，我认为虽然中国的经济水平在世界范围内跟发达国家比还不算好，但是玩 frp 的用户应该都不差这 10 块钱吧？平时少抽一盒烟，或少买瓶饮料，或少点份外卖，1个月在哪都能匀出这 10 块钱来。而且相比竞品，樱花算是便宜的了。
